msal/src/main/java/com/microsoft/identity/client/ISingleAccountPublicClientApplication.java

```diff
@@ -64,7 +63,11 @@
     ICurrentAccountResult getCurrentAccount() throws InterruptedException, MsalException;
 
     /**
-     * Allows a user to sign in to your application with one of their accounts.
+     * Allows a user to sign in to your application with one of their accounts. This method may only
```

Comment:
@shoatman I've added this to javadoc to clarify what the behavior of this method is